src/EA.Weee.Web/Areas/Producer/Mappings/ToViewModel/AddSignatoryAndCompleteMap.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace EA.Weee.Web.Areas.Producer.Mappings.ToViewModel
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.DirectRegistrant;
+    using EA.Weee.Core.Organisations;
+    using EA.Weee.Core.Shared;
+    using EA.Weee.Web.Areas.Producer.ViewModels;
+    using EA.Weee.Web.ViewModels.Shared;
+
+    public class AddSignatoryAndCompleteMap : IMap<SmallProducerSubmissionData, AppropriateSignatoryViewModel>
+    {
+        private readonly IMapper mapper;
+
```

Comment:
@stevelee-civica think we need unit tests for this one, sorry if I missed it

Reply:
added unit tests